workers/docker-worker/test/images/dind-test/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM        gliderlabs/alpine:3.2
-MAINTAINER  Jonas Finnemann Jensen <jopsen@gmail.com>
+FROM        alpine:latest
+MAINTAINER  Taskcluster <taskcluster-accounts+docker-maintainer@mozilla.com>
 
 # Install dependencies
 RUN         apk-install docker
```

Comment:
this will not run in latest alpine 
check this https://github.com/taskcluster/taskcluster/pull/5715/files 

Also maybe let's pin the version?

Reply:
Discussed via slack, we decided this is ok, as many other tests use `:latest`, and this way they are at least consistent.